docs/source/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Apollo Link
+sidebar_title: Introduction
+description: Apollo Link is a standard interface for modifying control flow of GraphQL requests and fetching GraphQL results.
+---
+
+This is the official guide for using Apollo Links in your Apollo app using the [apollo-link](https://github.com/apollographql/apollo-link) package. Apollo Link is a simple yet powerful way to describe how you want to execute a GraphQL operation, and what you want to do with the results. It is a convenient way to customize Apollo Client to the needs of your application while providing a small API to learn.
+
+Follow the repository on GitHub: [Apollo Link](https://github.com/apollographql/apollo-link).
+
+<h2 id="tutorials">Getting started</h2>
+
+If you're familiar with web development, but haven't tried GraphQL or Apollo before, we've got you covered. Here's a set of small tutorials and examples you can look at, and in just a few hours you'll be well on your way to being an expert GraphQL developer! Or, you can jump right in to [installing the library](initialization.html).
```

Comment:
This doesn't really fit, and I think the rest of this page is just the content from the react docs.

Reply:
I thought it would be valuable to copy this if it acts a landing page. I'll remove it for now though!